receiver/hostmetricsreceiver/hostmetrics_receiver.go

```diff
@@ -58,32 +66,96 @@ func NewHostMetricsReceiver(
 	hmr := &Receiver{
 		config:   config,
 		scrapers: scrapers,
+		consumer: consumer,
 	}
 
 	return hmr, nil
 }
 
-// Start begins scraping host metrics based on the OS platform.
+// Start initializes the underlying scrapers and begins scraping
+// host metrics based on the OS platform.
 func (hmr *Receiver) Start(ctx context.Context, host component.Host) error {
+	ctx, hmr.cancel = context.WithCancel(ctx)
+
 	go func() {
-		for _, scraper := range hmr.scrapers {
-			err := scraper.Start(ctx)
-			if err != nil {
-				host.ReportFatalError(err)
+		hmr.initializeScrapers(ctx, host)
+		hmr.startScrapers(ctx)
+	}()
+
+	return nil
+}
+
+// Shutdown terminates all tickers and stops the underlying scrapers.
+func (hmr *Receiver) Shutdown(ctx context.Context) error {
+	hmr.cancel()
+	return hmr.closeScrapers(ctx)
+}
+
+func (hmr *Receiver) initializeScrapers(ctx context.Context, host component.Host) {
+	bootTime, err := gopshost.BootTime()
+	if err != nil {
+		host.ReportFatalError(err)
+		return
+	}
+
+	startTime := pdata.TimestampUnixNano(bootTime)
+	for _, scraper := range hmr.scrapers {
+		err := scraper.Initialize(ctx, startTime)
```

Comment:
Consider to not pass the start time if we want this to be generic for other scrapers like Redis, etc. Also not that important as optimization since it happens once per lifetime of the scraper.